package.json

```diff
@@ -44,6 +44,6 @@
     "marked": "^0.3.12",
     "mustache": "^2.3.0",
     "newline-remove": "^1.0.2",
-    "throng": "^2.0.1"
+    "throng": "^4.0.0"
```

Comment:
The **Node** version should be higher than 4, so, given the *engines node* in the **package.json**, this update should be OK